VoxeUnity/Assets/Voxelmetric/Code/Core/Chunk.cs

```diff
@@ -427,21 +431,21 @@ private static void UpdateNeighborCount(Chunk chunk)
                 return;
 
             // Calculate how many neighbors a chunk can have
-            int maxNeighbors = 0;
+            byte maxNeighbors = 0;
             Vector3Int pos = chunk.Pos;
-            if (world.CheckInsideWorld(pos.Add(Env.ChunkSize, 0, 0)) && pos.x <= world.Bounds.maxX)
+            if (world.CheckInsideWorld(pos.Add(Env.ChunkSize, 0, 0)))
```

Comment:
These conditions are necessary.
Based on your settings you might want to have chunks at the visible world bounds created.

Reply:
Ah, I see.